### DISCUSSION TITLE
Bus stop and taxi rank "still here" check

### DISCUSSION THREAD

**Discussion by boothym** [streetcomplete/StreetComplete#4696] [Q&A] [answered]
Does StreetComplete ask "Is this still here?" for highway=bus_stop or amenity_taxi objects?

**Comment by matkoniecz** [comment, marked as answer]
From looking at https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/existence/CheckExistence.kt - no

For bus stops there are several quests where user may answer that it does not exist in note, asking about `amenity=taxi` may make sense - at least for nodes

I added it in https://github.com/streetcomplete/StreetComplete/commit/e49d807107dfa7e507bd46090675bd47eaa190aa - see https://overpass-turbo.eu/s/1pg8 for listing of objects being asked about with default settings

**Reply by FloEdelmann** [reply]
The quests about bin, bench and shelter are asked every 4 years (you can half that interval in the app settings).

**Reply by boothym** [reply]
Thanks for adding the taxi check.

**Reply by matkoniecz** [reply]
(hopefully converting into QA and marking as answered is fine - this way it is listed as handled on discussion list and I will not keep opening it when searching for something else with thought "hmm it seems fixable" and discovering that I fixed it months ago)
